src/styles/main.css

```diff
@@ -5,4 +5,18 @@ body {
   background-color: var(--color-primary);
   color: var(--color-secondary);
   font-family: var(--font-primary);
+}
+body::-webkit-scrollbar {
+  width: .8em;
+  height: 1em;
+}
+ 
+body::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+ 
+body::-webkit-scrollbar-thumb {
+  background-color: var(--color-accent);
+  outline: 1px solid none;
+  border-radius: 10px;
 }
```

Comment:
I did a quick search and looks like if we [add this line](https://stackoverflow.com/a/62910363/417806)
```css
html {
  scrollbar-color: var(--color-accent) var(--color-accent);
}
```

We can also get this working for FF too!  ✨
![Screen Shot 2022-10-26 at 8 14 55 PM](https://user-images.githubusercontent.com/895923/198162386-1a59f065-3dd7-49ad-831d-22fd7b437cba.png)

Reply:
if we add this it will turn the entire scrollbar var (--color-accent) on firefox
![test](https://user-images.githubusercontent.com/70280902/198225017-9a96bb7a-edf6-4854-b8c2-7b9fe4409d76.PNG)

Reply:
Oh right yeah, that makes sense when setting them both to the same value.  🤪 

Maybe it could be differences between MacOS / Windows as well?  ( think it looks like you may be on Windows, but correct me if I am wrong).  Reading the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color), I got closer on MacOS using these values, wonder how they look for you?
```css
html {
  scrollbar-color: var(--color-accent) var(--color-primary); /* first color is the thumb, second color is the track */
}
```

Not a big deal if not, but worth a shot!  🤞

Reply:
Also I suppose at some point when browsers implement this feature, we would want that un-prefixed version to exist so that it will start working on those browsers at that time.  We can consider the prefix solution to be more like a "fallback" solution in the interim.

Reply:
Alright i will add this
 html {
  scrollbar-color: var(--color-accent) var(--color-primary); /* first color is the thumb, second color is the track */
}
to the css and request a PR

Reply:
Sounds good.  You can just update this PR with and you don't have to include the comment, it was me mostly thinking out loud.

Oh, one favor, could you include this line in the file before the prefixed rules?  Should be good to go after that.

Thanks!

Reply:
scrollbar prefix firefox done